README.md

```diff
@@ -185,6 +185,13 @@ We could pass an instance of the `DNN` class to the `network_fn` parameter:
 build_model.network_fn = @DNN()
 ```
 
+or 
+
+```python
+# Inside "config.gin"
+build_model.network_fin=@DNN(num_outputs=15)
+```
+
 To use evaluated references, all of the referenced function or class's
```

Comment:
I find this sentence confusing, as we only need to specify some parameters?

Reply:
`DNN` defines default parameter values, and parameter value is also configured here by `gin.config` here .
Basicly , parameters might be overriding, and the priority is 
explicit keyword parameter values  => configured  => default parameter value

```python
    @gin.configurable()
    def _value(x):
      return x

    @gin.configurable
    def _add(a, b=1):
      return a + b

    config_str = """
      _add.a = 1
      _value.x=@_add()
    """
    gin.parse_config(config_str)
    self.assertEqual(2, _value())

    config_str = """
      _add.a = 1
      _value.x=@_add(a=2)
    """

    gin.parse_config(config_str)
    self.assertEqual(3, _value())

    config_str = """
      _add.a = 1
      _value.x=@_add(a=2, b=3)
    """

    gin.parse_config(config_str)
    self.assertEqual(5 ,_value())

```